data/json/recipes/chem/fuel.json

```diff
@@ -0,0 +1,108 @@
+[
+  {
+    "result": "chem_ethanol",
+    "type" : "recipe",
+    "byproducts": [ "water_clean" ],
+    "category": "CC_CHEM",
+    "subcategory": "CSC_CHEM_FUEL",
+    "skill_used": "cooking",
+    "difficulty": 5,
+    "time": 30000,
+    "book_learn": [ [ "textbook_chemistry", 5 ], [ "brewing_cookbook", 4 ] ],
+    "qualities": [
+        { "id": "BOIL", "level": 2, "amount": 1 },
+        { "id": "BREW", "level": 2, "amount": 1 }
+     ],
+     "tools": [
+       [ [ "fire", -1 ], [ "hotplate", 30 ], [ "toolset", 30 ] ],
+       [ [ "thermometer", -1 ] ]
+     ],
+     "components": [ [
+       [ "vodka", 14 ],
+       [ "whiskey", 14 ],
+       [ "single_malt_whiskey", 14 ],
+       [ "brandy", 14 ],
+       [ "rum", 14 ],
+       [ "gin", 14 ],
+       [ "tequila", 14 ],
+       [ "bum_wine", 56 ],
+       [ "fruit_wine", 56 ],
+       [ "moonshine", 14 ],
+       [ "mixed_alcohol_strong", 14 ]
+    ] ]
+  },
+  {
+    "result": "denat_alcohol",
+    "type" : "recipe",
+    "byproducts": [ "water_clean" ],
+    "category": "CC_CHEM",
+    "subcategory": "CSC_CHEM_FUEL",
+    "skill_used": "cooking",
+    "difficulty": 3,
+    "time": 30000,
+    "book_learn": [ [ "textbook_chemistry", 4 ], [ "brewing_cookbook", 3 ] ],
+    "qualities": [
+        { "id": "BOIL", "level": 1, "amount": 1 },
+        { "id": "BREW", "level": 1, "amount": 1 }
```

Comment:
> How are you going to concentrate alcohol with just a pot and some heat?

I missed out the requirement for a chemistry kit.

> And how would it get denaturated in the process?

Failure to remove the small quantities methanol (which requires a thermometer and much more careful operation) result it also being concentrated and the resultant product being contaminated with it and therefore poisonous. This was the major problem during prohibition (presumably a bit before all of our times).

Denatured alcohol is just ethanol with a sufficiently large quantity of methanol added to make economic removal of it impractical (caveat: some also contains dye and bittering agent)